en/components/tree.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Tree Component - Design System Component
+_description: The Tree Component Symbol is used as a visual representation of the hierarchical reletaionship between categories, presented in a list structure.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits, Figma, Figma to Angular, Export code from Figma, Figma HTML, Figma to HTML, Figma UI kits
+---
+
+## Tree
+The Tree component is used as a visual representation of the hierarchical reletaionship between categories, presented in a list structure.
+Tree is visually identical to the [Ignite UI for Angular Tree Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/tree.html)
+
+## Tree Demo
+
+<img class="responsive-img" src="../images/tree_demo.png" srcset="../images/tree_demo@2x.png 2x" />
+
+## Tree Density
+
+The Tree component supports three display densities - Comfortable, Cosy and Compact. In Figma you can switch between the three by changing the value of the "Display Density" property from the properties panel, while in Sketch you need to choose the display density prior to inserting the symbol.
+
+<img class="responsive-img" src="../images/tree_density_comfortable.png" srcset="../images/tree_density_comfortable@2x.png 2x" />
+<img class="responsive-img" src="../images/tree_density_cosy.png" srcset="../images/tree_density_cosy@2x.png 2x" />
+<img class="responsive-img" src="../images/tree_density_compact.png" srcset="../images/tree_density_compact@2x.png 2x" />
+
+## Tree Type
+
+We support two types of Tree component: Basic and Checkbox, where you have a Checkbox component after the Expand Icon. In Figma you can switch between the two types using the "Checkbox" boolean property from the properties panel, while in Sketch you have to choose the type prior to inserting the symbol, after choosing the display density.
```

Comment:
We support two types of Tree component ... - "component" maybe should be "components"

Reply:
I left it component for now, because we have two types for a single component. I'll leave the comment for Svilen to double check :)